ding/reward_model/rnd_reward_model.py

```diff
@@ -138,14 +134,31 @@ def estimate(self, data: list) -> None:
 
             rnd_reward = rnd_reward.to(data[0]['reward'].device)
             rnd_reward = torch.chunk(rnd_reward, rnd_reward.shape[0], dim=0)
+        """NOTE:
+        Following normalization approach to extrinsic reward seems be not reasonable,
+        because this approach compresses the extrinsic reward magnitude, resulting in less informative reward signals.
+        """
+        # rewards = torch.stack([data[i]['reward'] for i in range(len(data))])
+        # rewards = (rewards - torch.min(rewards)) / (torch.max(rewards) - torch.min(rewards))
+
+        # TODO(pu): how to set intrinsic_reward_weight and intrinsic_reward_rescale automatically?
```

Comment:
why are there still this TODO

Reply:
still need to manually set intrinsic_reward_rescale now.